components/rhs_thread/rhs_thread.tsx

```diff
@@ -172,6 +172,19 @@ export default class RhsThread extends React.Component<Props, State> {
         }
     }
 
+    private userIsAtLastPost = () => {
+        const postReplies = (this.rhspostlistRef.current as HTMLElement)?.childNodes;
+        const rhsPostListRect = (this.rhspostlistRef.current as HTMLElement)?.getBoundingClientRect();
+        const offset = 24;
+
+        if (postReplies) {
+            const postRepliesArray = Array.from(postReplies);
+            return ((postRepliesArray[postRepliesArray.length - 1] as HTMLElement).offsetTop + offset) + rhsPostListRect.top === rhsPostListRect.bottom;
```

Comment:
I'm not sure about equality here, if the list is scrolled a few pixels up it shouldn't scroll although the last post is almost entirely visible? 0/5 on this, but I think we should at least consider it :)

Reply:
Makes sense to address it with UX with the new implementation but for this I guess the user will be scrolled down just a little bit when the new message is incoming? I did a few tests by tweaking the values and it all seemed ok to me but additional opinions are welcome!